services/app-web/src/components/DataDetail/DataDetailContactField/DataDetailContactField.tsx

```diff
@@ -5,9 +5,9 @@ import {
 } from '../../../common-code/healthPlanFormDataType'
 import { getActuaryFirm } from '../../SubmissionSummarySection'
 import { DataDetailMissingField } from '../DataDetailMissingField'
+import { ActuaryContact as GQLActuaryContact } from '../../../gen/gqlClient'
 
-type Contact = ActuaryContact | StateContact
-
+type Contact = ActuaryContact | StateContact | GQLActuaryContact
```

Comment:
@macrael this was another issue where the graphql types and common code types don't translate well `Type 'Maybe<string> | undefined' is not assignable to type 'string | undefined'.`  

There are many ways to work around this using null coalescing checks, adding to the types (like I'm doing here), manually asserting its a same type but ideally I wanted to map these two types together using the `codegen.yml.` similar to what we do with Rate Revision. Couldn't get this to actually fixing the issue though. Any suggestions for the future? This kind of types issue conflict will keep happening on the frontend

Reply:
I think that GQL using Maybe is something that can be configured in codegen.yml so that's probably worth looking into